### ISSUE TITLE
Test ssh commands

### ISSUE THREAD

**Issue by flyingarg** [flyingarg/bgppp#16]
New commands added to ssh - 
linkadd : Add a link
linkls: Displays list of links
ifls: Displays list of interfaces on the routers
ifadd: Add interfaces on the router
ifdel: Delete interfaces on the router

We need to test and debug these commands

**Comment by Josan-Coba** [comment]
##### Interface commands test:
1. `ifls` seems to work correctly.
2. `ifadd` allows to overwrite any existing interface without prompting the user.
3. `ifdel` shows weird behaviour:
 - One can `ifdel` the same interface being used for the current ssh connection, actually making the ssh command to freeze.
 - `ifdel`'ing any other interface will prompt a "success" message, but if a `ifls` command follows, the interface will remain there. `Linkadd`'ing a deleted interface won't generate any prompt but will actually freeze the ssh command line. After halting the command line, re-logging and running a `linkls` command, the previously deleted interface and the created link over it will be shown there.

##### Link commands test:
1. `linkls` seems to work correctly.
2. `linkadd` presents some issues:
 - Running `linkadd` without parameters prompts that the correct call is `linkadd sourceaddress destinyaddress`. However, `linkadd` fails if the first argument is an address, and works correctly when passing an interface name.
 - `linkadd` will correctly be executed with a non-existant destiny address.

**Comment by flyingarg** [comment]
I agree, these are all valid bugs. Could you create bugs in the issue tracker, if u want, you can assign them to yourself, or we can make assignments later. 
All the commands have their own methods in <code>remote/SSHCommands</code>

**Comment by Josan-Coba** [comment]
Is there a known way to check the SSH interface being used from the code? (In order to avoid deleting it)

**Comment by flyingarg** [comment]
hi, i did not understand the question.

**Comment by Josan-Coba** [comment]
As specified, right now you can ifdel the same interface you used to connect (You can do "SSH 10.50.0.1" and then "ifdel 10.50.0.1"). I'm trying not allow that behaviour, but I don't know how to check, from the code of SSHCommands class, what's the IP used to connect. Any ideas?
